js/views/TooltipItemView.js

```diff
@@ -129,6 +129,11 @@ export default class TooltipItemView extends Backbone.View {
     this.remove();
   }
 
+  onClick() {
```

Comment:
I can't see where this function is being called from. There's no click event listener in this view.

Reply:
It's called on line 36:

`$(document).on('click', '[data-tooltip-id]', this.onClick);`

Reply:
Different file.

Reply:
Dah, you're right. Must have still been there from an earlier attempt to fix it.